server.js

```diff
@@ -198,28 +198,33 @@ function Server (opts) {
           if (keys.length > 0) activeTorrents++
 
           keys.forEach(function (peerId) {
-            if (!allPeers.hasOwnProperty(peerId)) {
-              allPeers[peerId] = {
-                ipv4: false,
-                ipv6: false,
-                seeder: false,
-                leecher: false
-              }
-            }
             // Don't mark the peer as most recently used for stats
             var peer = peers.peek(peerId)
-            if (peer.ip.indexOf(':') >= 0) {
-              allPeers[peerId].ipv6 = true
-            } else {
-              allPeers[peerId].ipv4 = true
-            }
-            if (peer.complete) {
-              allPeers[peerId].seeder = true
-            } else {
-              allPeers[peerId].leecher = true
+
+            // The peer could be evicted at this point
+            if (typeof peer !== 'undefined') {
```

Comment:
Fixed in https://github.com/feross/bittorrent-tracker/commit/f2786cde96e90f4f16b4c843dbcea590c4675da4